es6/mobify-es6.yml

```diff
@@ -188,7 +185,6 @@ rules:
   space-infix-ops: error
   space-unary-ops: error
   spaced-comment: error
-  strict: error
```

Comment:
IMHO I rarely see 'use strict' in any files at Mobify, and even though it's unnecessary I don't think it's harmful to have it in a file where it's not needed. I'd rather let a few unnecessary 'use strict' definitions go through than encourage disabling linting rules.